calicoctl/calico_ctl/node.py

```diff
@@ -574,6 +575,9 @@ def node_stop(force):
     Stop the Calico node.  This stops the containers (calico/node and
     calico/node-libnetwork) that are started by calicoctl node.
     """
+    # The command has to be run as root to stop the calico-node service
```

Comment:
I can answer that: we've baked in `systemd` for when we're running in rkt.  Take a look at how `calicoctl node --runtime=rkt` works.